app/src/main/java/com/cornellappdev/uplift/ui/components/general/NoClasses.kt

```diff
@@ -31,22 +33,23 @@ fun NoClasses() {
         )
         Spacer(modifier = Modifier.height(12.dp))
         Text(
-            text = "No classes today",
+            text = if (comingSoon) "Classes coming soon!" else "No classes today",
             fontWeight = FontWeight(700),
             fontSize = 24.sp,
             lineHeight = 29.26.sp,
             textAlign = TextAlign.Center,
             color = PRIMARY_BLACK,
             fontFamily = montserratFamily
         )
-        Text(
-            text = "Relax with some tea or play a sport",
-            fontWeight = FontWeight(400),
-            fontSize = 17.sp,
-            lineHeight = 17.07.sp,
-            textAlign = TextAlign.Center,
-            color = PRIMARY_BLACK,
-            fontFamily = montserratFamily
-        )
+        if (!comingSoon)
+            Text(
+                text = "Relax with some tea or play a sport",
```

Comment:
aha this is some funny text

Reply:
u ate

Reply:
lol tell that to design, i just typed it in for them